addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -298,7 +313,16 @@
                 QWeb.render('slide.slide.quiz.validation', {'widget': this})
             );
         },
-
+        /*
+        * Toggle additional resource info box
+        * 
+        * @private
+        * @param {Boolean} show - Whether show or hide the information
+        */
+        _toggleAdditionalResourceInfo: function(show) {
+            const resourceInfo = document.getElementsByClassName('o_wslides_js_lesson_quiz_resourceinfo')[0];
+            show && resourceInfo ? resourceInfo.classList.remove('d-none') : resourceInfo.classList.add('d-none');
```

Comment:
Is the "&& resourceInfo" necessary ? Anyway if it does not exist, won't it crash ?

Reply:
Correct. It should be
```suggestion
            resourceInfo && (show ? resourceInfo.classList.remove('d-none') : resourceInfo.classList.add('d-none'));
```